src/guide/migration/attrs-includes-class-style.md

```diff
@@ -0,0 +1,69 @@
+---
+title: $attrs includes class & style
+badges:
+  - breaking
+---
+
+# `$attrs`includes `class` & `style` <MigrationBadges :badges="$frontmatter.badges" />
```

Comment:
There should probably be a space between $attrs and includes, though I'm not sure it will actually make any difference to how this is rendered with the current theme.